homeassistant/components/homematic/const.py

```diff
@@ -111,7 +113,7 @@
         "IPThermostatWall2",
         "IPRemoteMotionV2",
         "HBUNISenWEA",
-        "IPWMotionDection",
```

Comment:
In case anybody is wondering: yes, this really has been removed in the pyhomematic code.

Reply:
Does this constitute a breaking change?

Reply:
No, `PresenceIPW` (one line below) is just the new name.